src/kernel_types.rs

```diff
@@ -392,3 +392,22 @@ pub fn en_wit_send_error_kind(kind: SendErrorKind) -> wit::SendErrorKind {
         SendErrorKind::Timeout => wit::SendErrorKind::Timeout,
     }
 }
+
+// python Requests and Responses encode the Request `code`
+//  and the Response `output` in the `lazy_load_blob`
+#[derive(Debug, Serialize, Deserialize)]
+pub enum PythonRequest {
+    Run,
+}
+
+#[derive(Debug, Serialize, Deserialize)]
+pub enum PythonResponse {
+    Run,
+    Err(String),
+}
+
+#[derive(Debug, Serialize, Deserialize)]
+pub enum MessageType {
+    Request,
+    Response,
+}
```

Comment:
The point of python executer being a runtime extension is so we don't have to put its types and API into the kernel, so we need to remove these and keep them somewhere else. Perhaps in a library shared by the runtime ext repo. Anywhere but here!